radix-engine/src/system/system_modules/costing/fee_table.rs

```diff
@@ -564,11 +564,23 @@ impl FeeTable {
         // Based on  `test_crypto_scrypto_verify_secp256k1_ecdsa_costing`
         //   instructions_cnt = 464236 (input is always 32 bytes long)
         //   Lets round:
-        //     444420.94242 -> 446000
-        let instructions_cnt = 446000;
+        //     464236 -> 500000
```

Comment:
I don't mind this at all, but purely out of interest, why are we rounding this one up a lot more than `ed25519_verify_cost` ?

Reply:
I think I have overlooked it. Was working on `Secp256k1` key recover and saw that other `Secp256k1`-related methods needs updating, so I've updated it. But forgot about `Ed25519`. I think it might be updated too for consistency.
Thanks for spotting it!